WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -317,6 +317,12 @@ private void validateAndStyleUsername(Collection<String> usernames, final Valida
         }
 
         if (usernamesToCheck.size() > 0) {
+
+            if (!NetworkUtils.checkConnection(getActivity())) {
```

Comment:
sure thing! done here https://github.com/wordpress-mobile/WordPress-Android/pull/4335/commits/157eca164ab2029dcf80487af4ad9a9cd4700148 - your suggestion seems to work well 👍